v2/common/src/test/java/com/google/cloud/teleport/v2/io/WindowedFilenamePolicyTest.java

```diff
@@ -131,21 +131,33 @@ public void testWithDynamicDirectory() throws IOException {
     when(window.maxTimestamp()).thenReturn(windowEnd);
     when(window.start()).thenReturn(windowBegin);
     when(window.end()).thenReturn(windowEnd);
-
+    
+    // Directory with "mm" specified
     WindowedFilenamePolicy policy =
         WindowedFilenamePolicy.writeWindowedFiles()
             .withOutputDirectory("gs://test-bucket-mm/YYYY/MM/dd/HH:mm")
             .withOutputFilenamePrefix("output")
             .withShardTemplate("-SSS-of-NNN");
+    
+    // Directory with "dd" specified
+    WindowedFilenamePolicy policy_with_dd =
+        WindowedFilenamePolicy.writeWindowedFiles()
+            .withOutputDirectory("gs://test-bucket-dd/YYYY/MM/dd/HH:mm")
```

Comment:
This is a good test. Maybe we could really take this to the next level by having all fields if we want to extrapolate 

e.g.
`.withOutputDirectory("gs://test-bucket-YYYY-MM-dd_HH_mm/YYYY/MM/dd/HH:mm")`

we expect the bucket to stay intact

Reply:
good idea. in that case i'll just modify the original test to include this (see latest commit)